app/services/DataStoreService.scala

```diff
@@ -50,13 +50,13 @@ class DataStoreService @Inject() (httpClient: HttpClientV2, metricsReporter: Met
         }
         .recover { case e =>
           log.error(s"DATASTORE-E-EORI-HISTORY-ERROR: ${e.getClass.getName}")
-          emptyEoriHistory
+          Seq(EoriHistory(emptyString, None, None))
         }
     }
   }
 
-  def getEmail(eori: EORI)(implicit hc: HeaderCarrier): Future[Either[EmailResponses, Email]] = {
-    val dataStoreEndpoint = s"${appConfig.customsDataStore}/eori/$eori/verified-email"
+  def getEmail()(implicit hc: HeaderCarrier): Future[Either[EmailResponses, Email]] = {
```

Comment:
Better to remove redundant () after method name. Same for below methods

getCompanyName()
getOwnCompanyName()
getXiEori()
getCompanyAddress()